edx_rest_api_client/client.py

```diff
@@ -5,6 +8,36 @@
 
 
 class EdxRestApiClient(slumber.API):
+
+    @classmethod
+    def get_oauth_access_token(cls, url, client_id, client_secret):
+        """ Retrieves OAuth 2.0 access token using the client credentials grant.
+        :param url: Oauth2 access token endpoint
+        :param client_id: client ID
+        :param client_secret: client secret
+        :return: (str, datetime) -- (access token, expiration datetime)
+        """
+        now = datetime.datetime.now(UTC)
+
+        response = requests.request(
+            "post",
+            url,
+            data={
+                "grant_type": "client_credentials",
+                "client_id": client_id,
+                "client_secret": client_secret,
+            }
+        )
+
+        response = response.json()
+
+        access_token = response.get('access_token')
+        expires_in = response.get('expires_in')
```

Comment:
What do you think of accessing these keys directly (using square brackets)? If they're missing, I think it'd be in our interest to fail hard.